db/seedfiles/translations/en/me/shared.rb

```diff
@@ -28,7 +28,7 @@
     "en.has_job_income_section_three" => "We only need an exact start date if this income began this year. If it began in a prior year and you don’t remember the exact start date, you can enter the first day of the current year as the start date.",
     "en.has_job_income_section_four" => "Only enter an end date for income in the ‘End Date’ field if this income has ended or is expected to end on a certain date. Otherwise leave this field blank.",
     "en.has_self_employment_income" => "<b>Is this person self-employed? </b> <br> <br> Select ‘yes’ if this person owns a business or receives a federal form 1099 from any employer. We need to know about any income this person receives as an independent contractor or from a business they own.",
-    "en.has_unemployment_income" => "<b>Did this person receive unemployment income at any point in 2021? </b> <br> <br> Select ‘yes’ if this person received one or more types of unemployment income listed below at any point in 2021, even if they no longer receive this income. <br> <ul> <li> Regular unemployment compensation </li> <li> Pandemic Emergency Unemployment Compensation (PEUC) </li> <li> Extended unemployment compensation </li><li> Pandemic Unemployment Assistance (PUA) </li></ul> <br> We need to know about any unemployment income this person received in 2021 to check if they qualify for lower monthly premiums under the <a href='https://www.dchealthlink.com/coronavirus/americanrescueplan' target='_blank' rel='noopener noreferrer'>American Rescue Plan</a>.",
+    "en.has_unemployment_income" => "<b>Did this person receive unemployment income at any point in %{application_applicable_year}? </b> <br> <br> Select ‘yes’ if this person received one or more types of unemployment income listed below at any point in %{application_applicable_year}, even if they no longer receive this income. <br> <ul> <li> Regular unemployment compensation </li> <li> Pandemic Emergency Unemployment Compensation (PEUC) </li> <li> Extended unemployment compensation </li><li> Pandemic Unemployment Assistance (PUA) </li></ul> <br> We need to know about any unemployment income this person received in %{application_applicable_year} to check if they qualify for lower monthly premiums under the <a href='https://www.dchealthlink.com/coronavirus/americanrescueplan' target='_blank' rel='noopener noreferrer'>American Rescue Plan</a>.",
```

Comment:
@saipraveen18 don't we have to pass in the application assistance year where this is being used?